lib/manager/composer/range.ts

```diff
@@ -42,7 +42,7 @@ export function getRangeStrategy(config: RangeConfig): RangeStrategy {
     logger.trace({ dependency: depName }, 'Pinning app require');
     return 'pin';
   }
-  if (isComplexRange) {
+  if (isComplexRange || ['typo3-cms-extension'].includes(composerJsonType)) {
```

Comment:
It's probably too many changes here but personally I'd move this list to a separate constant instead of having it inline. (Same for the longer list above.)

Reply:
I just adhere to the existing code style.

Reply:
I know and keeping the change to a minimum is a good thing. :-)